packit/local_project.py

```diff
@@ -442,6 +444,14 @@ def create_branch(
 
         return head
 
+    def _fetch_as_branch(
+        self, remote_ref: str, local_ref: str, local_branch: str
+    ) -> None:
+        remote = self.remote or "origin"
+        self.git_repo.remotes[remote].fetch(f"{remote_ref}:{local_ref}")
+        self.git_repo.create_head(local_branch, f"{remote}/{local_branch}")
+        self.git_repo.branches[local_branch].checkout()
```

Comment:
this is pretty nasty to do checkout in a method call branch, I would personally not expect that - how about doing this operation in the merge_pr method?

Reply:
more likely to update name and docs, since the checkout is done from both `checkout_pr` and `merge_pr`; fine with you? :)

Reply:
changed my mind, I don't think I need the PR to be checked out :thinking: